test/Test/Fixtures.hs

```diff
@@ -74,9 +74,10 @@ projectRevision =
     }
 
 sourceUnits :: NE.NonEmpty SourceUnit
-sourceUnits =
-  NE.fromList
-    [ SourceUnit
+sourceUnits = unit NE.:| []
```

Comment:
IMO, it's fine to import `:|` unqualified into the module namespace. I think it reads nicer that way, and I'm not aware of any operators that conflict with it.

Reply:
Yeah but that changes the import.  I didn't really care, and I don't think this is any less readable.  I'm planning a cleanup soon to take care of inconsistent imports.

Reply:
I only did this because there's no `NE.singleton`... yet.